### PR TITLE
framework/media: fix bug for prepareasync

### PR DESCRIPTION
**Pull request by jsdosa**
Fixed a timing issue between threads.

Signed-off-by: jsdosa <jsdosa@gmail.com>

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [95ed7f2a6c8c0c7fadc4ab3af9e9b716e657b181] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [95ed7f2a6c8c0c7fadc4ab3af9e9b716e657b181] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [3f5921e6d70a76911024527ce51f002eb23381f5] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [3f5921e6d70a76911024527ce51f002eb23381f5] - Code Rule Check OK.

File: framework/src/media/MediaPlayerImpl.cpp [hunk 1]
```diff
@@ -735,10 +738,10 @@ void MediaPlayerImpl::notifyAsync(player_event_t event)
 {
 	LOG_STATE_INFO(mCurState);
 
-	if (mCurState != PLAYER_STATE_PREPARING) {
-		meddbg("%s Fail : invalid state\n", __func__);
-		LOG_STATE_DEBUG(mCurState);
-		return notifyObserver(PLAYER_OBSERVER_COMMAND_ASYNC_PREPARED, PLAYER_ERROR_INVALID_STATE);
+	if (mCurState != PLAYER_STATE_PREPARING) {
```
Comment (junmin-kim) [inline framework/src/media/MediaPlayerImpl.cpp]: remove tabs

Comment (jsdosa) [reply framework/src/media/MediaPlayerImpl.cpp]: yes thanks

**Comment by seinfra** [comment]
Target : [7baf0f5c52ed1568fe76d51e2a447b604922fabf] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [7baf0f5c52ed1568fe76d51e2a447b604922fabf] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [8334a0eca22001f13691042afc507b8279ac1ff6] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [8334a0eca22001f13691042afc507b8279ac1ff6] - Code Rule Check OK.

**Pull request merged**